packages/authenticator/amplify_authenticator/lib/src/widgets/social/social_button.dart

```diff
@@ -3,24 +3,26 @@
 
 import 'dart:math';
 
-import 'package:amplify_auth_cognito/amplify_auth_cognito.dart';
 import 'package:amplify_authenticator/src/l10n/auth_strings_resolver.dart';
 import 'package:amplify_authenticator/src/state/authenticator_state.dart';
+import 'package:amplify_authenticator/src/utils/get_social_auth_providers.dart';
 import 'package:amplify_authenticator/src/utils/list.dart';
 import 'package:amplify_authenticator/src/widgets/button.dart';
 import 'package:amplify_authenticator/src/widgets/component.dart';
 import 'package:amplify_authenticator/src/widgets/social/social_icons.dart';
-import 'package:aws_common/aws_common.dart';
+import 'package:amplify_core/amplify_core.dart';
 import 'package:flutter/foundation.dart';
 import 'package:flutter/material.dart';
 
 class SocialSignInButtons extends StatelessAuthenticatorComponent {
   const SocialSignInButtons({
     super.key,
-    required this.providers,
+    this.buttons,
   });
 
-  final List<AuthProvider> providers;
+  /// [SocialSignInButton]s to render in a column with space between. If omitted,
+  /// will render buttons based on social providers from Amplify config.
+  final List<SocialSignInButton>? buttons;
```

Comment:
Maybe we need the `includeDefaultSocialProviders` flag here so that the following works:

```dart
SocialSignInButtons(
  buttons: [AuthProvider.custom('abcd')],
  includeDefaultSocialProviders: true,
)
```